loadbalancer/templates/nginx.conf.conf

```diff
@@ -123,4 +118,34 @@ http {
     }
 
 	}
+
+  server {
+      listen 80;
+      server_name ${AMUI_TEST_DOMAIN};
+
+      proxy_redirect      off;
+      proxy_set_header    Host $host;
+      proxy_set_header    X-Real-IP $remote_addr;
+      proxy_set_header    X-Forwarded-For $proxy_add_x_forwarded_for;
+
+    error_page 502 /502Accessmanagement.html;
+
+		location = / {
+        proxy_pass          http://accessmanagementbff/accessmanagment/;
+		}
+
+    location / {
+        #Support using Local js, when a cookie value is set
+        sub_filter_once off;
+        sub_filter 'https://altinncdn.no/toolkits/altinn-app-frontend/3/' $LOCAL_SUB_FILTER;
+        proxy_pass          http://accessmanagementbff/;
+        error_page 502 /502Accessmanagement.html;
+        proxy_cookie_domain altinn3local.no local.altinn.cloud;
```

Comment:
`proxy_cookie_domain` should not be required here, and I’m not sure about the `sub_filter`

Reply:
I'm not sure about it either. @TheTechArch @tjololo Do you have any comments on this?

Reply:
You dont need it. Same with Sub filter also I belive.  Test without

Reply:
@allinox You can remove the sub_filter and proxy_cookie_domain here and test it out before committing the changes to this branch